docs/user-guides/finetuner.md

```diff
@@ -174,10 +175,6 @@ executors:
     replicas: 1
 ```
 
-```{warning}
```

Comment:
Do not remove it yet. Change it to list 4 models not yet supported by Finetuner (and maybe M-CLIP)